feat: Add a modal to NSFW sticker packs

romainricard opened the pull request:
Add a modal hiding the content of NSFW packs, and asks the user to confirm that they really want to see the pack. This aims to better warn the user that a pack is NSFW.

As this is a minor feature, I tried to keep the code simple.

![image](https://user-images.githubusercontent.com/7778898/76908725-b134a300-68a9-11ea-963d-c4995473172d.png)

darkobits commented on src/components/pack/NsfwModal.tsx:
```diff
@@ -0,0 +1,74 @@
+import {styled} from 'linaria/react';
```
Noticed mixed indentation between 2 spaces and 4 spaces in this file. We use 2 spaces in the rest of the repo. I've opened https://github.com/signalstickers/signalstickers/pull/152 which adds an `.editorconfig` file to the repository which should help to automatically prevent these issues in the future.

romainricard commented on src/components/pack/NsfwModal.tsx:
```diff
@@ -0,0 +1,74 @@
+                        </div>
+                        <div className="modal-body">
+                            <p>
+                                This pack has been marked <i>Not Safe For Work</i> (<a href="https://www.urbandictionary.com/define.php?term=NSFW" target="_blank" rel="noreferrer">NSFW</a>).
```
AFAIK `noreferrer` is _supposed_ to act as `noopener` + no referrer. Works on FF, but to be sure, I'll also include `noopener`!

romainricard commented:
Fixed, thanks! @darkobits don't forget to `git diff --ignore-space-change src/components/pack/NsfwModal.tsx`, as I've also fixed the indentation problem.
